lib/cassette/cache.rb

```diff
@@ -8,28 +8,6 @@
 
 module Cassette
   module Cache
-    def self.backend
-      @backend ||= begin
-        if defined?(::Rails) && ::Rails.cache
-          ::Rails.cache
-        elsif defined?(::ActiveSupport::Cache::MemoryStore)
-          ActiveSupport::Cache::MemoryStore.new
-        else
-          NullStore.new
-        end
-      end
-    end
-
-    def self.backend=(backend)
```

Comment:
Can't we delegate the writer just like we are delegating the reader?

Reply:
You're right! I've just added this writer and now I could keep the compatibility. Could you review again pls?